src/TwitchClient.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const axios = require('axios');
+const moment = require('moment');
+const TwitchEmbed = require('./embeds/TwitchEmbed');
+
+/**
+ * Manages Twitch API
+ */
+class TwitchClient {
```

Comment:
idea: treat it as an emitter and fire an event, allowing a notifier, or build it into the twitch client, to push the embed to the notifier system and have that fire the events out to any subscriber instead of a single channel

Reply:
Yeah I can look into that. 

Does this go hand in hand with your comment above about "trackable"?

Do you have an example of what you mean by this in the code?

Reply:
yeah.... take a look at the [RSS Emitter](https://github.com/WFCD/genesis/blob/master/src/notifications/FeedsNotifier.js)

Reply:
@TobiTenno I think I am close to having this working, but it looks like there are 0 channels in the Broadcaster. How do I setup a channel for testing? Is this a custom setting of some kind within the discord server?

Thanks!

Reply:
https://github.com/WFCD/genesis/blob/master/src/notifications/Broadcaster.js#L60

channels.length is 0 for me

Reply:
Looks like it has something to do with the database.

This is what my broadcast call looks like, so i'm guessing i need something in the database that links a channel to 'twitch'?

// broadcast it!
platforms.forEach((platform) => {
 this.broadcaster.broadcast(twitchEmbed, platform, 'twitch');
});

Reply:
So I got it to work by running this SQL manually against my mysql database. But I am curious, of the right way to set this up:

-- 4982... is my #general channel on my discord server
insert into type_notifications values (498248759872847904, 'twitch', 1)

Reply:
Yo, sorry I couldn't answer for a while. Yeah, you'll need to add the twitch event as a trackable, track it, and you shoulld be getting them. Looks like you got it inserted. You can also use the commands to do it (which is what people using it would use normally)

Reply:
What’s the command? Does it work when running locally?

Reply:
`/track twitch`